pulp_container/app/viewsets.py

```diff
@@ -643,7 +642,6 @@ class ContainerDistributionViewSet(BaseDistributionViewSet):
                 "effect": "allow",
                 "condition": [
                     "has_model_or_obj_perms:container.change_containerdistribution",
-                    "has_namespace_obj_perms:container.manage_namespace_distributions",
```

Comment:
why are we removing them here?

Reply:
I thought to leave manage_namescpace_distributions only when we add new ones or remove existing. For update operations it feels like it is enough to grant to the user this specific object-level perms. WDYT?

Reply:
Yeah, it's hard. I thought we might need to replicate all the distribution permissions on the namespace level.
So when you are a consumer of that namespace, you can pull from all repos therein, including future ones.

Reply:
i am hoping this will be managed by the created groups and assigned to those set of permissions.